src/entities/task/ui.tsx

```diff
@@ -0,0 +1,17 @@
+import { Task } from "@entities/task/model";
+
+const TaskItem = ({ id, title }: Omit<Task, 'completed'>) => (
+    <div>{title}</div>
+)
+
+export const TaskList = ({ tasks }: { tasks: Task[] }) => {
+    return (
+        <>
```

Comment:
You can wrap it to `div`, because it's correct to have particular parent node for items